scala/scala.bzl

```diff
@@ -164,6 +187,17 @@ def _write_test_launcher(ctx, jars):
       output=ctx.outputs.executable,
       content=content)
 
+def _collect_srcjars(targets):
+  srcjars = set()
+  for target in targets:
+    print(target) #TODO remove
+    #TODO perhaps we want to unify the naming?
+    if hasattr(target, "srcjar"):
```

Comment:
I think `srcjar` is only used in `java_import` and we don't really have an analog of that, so I think we can skip it.